tests/integration/components/head-tag-test.js

```diff
@@ -58,7 +58,6 @@ module('Integration | Component | head tag', function(hooks) {
       'src',
       'property',
       'itemprop',
-      'id'
```

Comment:
This is removed because it makes no sense. Further down in the test this ID is being overriden.

Reply:
https://github.com/ronco/ember-cli-meta-tags/pull/80

On interesting

Reply:
I saw that, but I still don't get it. When we render the component on this very same test with `await render(hbs`{{head-tag headTag=headTag id='test-comp'}}`);` we are passing `id='test-comp'`, which precisely overwrites the ID in `headTag`.

What I don't understand is how this even worked to begin with.